packages/unitables/src/bee/UnitablesBeeTable.tsx

```diff
@@ -32,12 +32,13 @@ import * as React from "react";
 import { useCallback, useMemo } from "react";
 import * as ReactTable from "react-table";
 import { UnitablesColumnType } from "../UnitablesTypes";
-
 import "@kie-tools/boxed-expression-component/dist/@types/react-table";
 import { ResizerStopBehavior } from "@kie-tools/boxed-expression-component/dist/resizing/ResizingWidthsContext";
 import { AutoField } from "@kie-tools/uniforms-patternfly/dist/esm";
 import { useField } from "uniforms";
 import { AUTO_ROW_ID } from "../uniforms/UnitablesJsonSchemaBridge";
+import get from "lodash/get";
```

Comment:
Shouldn't we be doing `_.get`? I mean, it's weird to have a `get` floating around :D

Reply:
As I'm using the default export, I can change the name of it for what we want. Maybe `getObjectValueByPath` is a better name.